src/sentry/constants.py

```diff
@@ -256,7 +256,7 @@ def get_all_languages() -> List[str]:
     ]
 )
 
-SCHEMA_FORM_ACTIONS = frozenset(
+SENTRY_APP_ACTIONS = frozenset(
```

Comment:
This name SCHEMA_FORM_ACTIONS isn't descriptive, nor helpful to identify what the action really is.